code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -148,19 +148,19 @@ There are several things that need to be remembered:
 	update_mutant_bodyparts()
 
 /mob/living/carbon/human/update_worn_id()
-	remove_overlay(ID_LAYER)
+	//remove_overlay(ID_LAYER) SKYRATEDIT - PDA-B-GONE
 
 	if(client && hud_used)
 		var/atom/movable/screen/inventory/inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_ID) + 1]
 		inv.update_icon()
 
-	var/mutable_appearance/id_overlay = overlays_standing[ID_LAYER]
+	// var/mutable_appearance/id_overlay = overlays_standing[ID_LAYER] SKYRATEDIT - PDA-B-GONE
 
 	if(wear_id)
 		var/obj/item/worn_item = wear_id
 		update_hud_id(worn_item)
-		var/icon_file = 'icons/mob/clothing/id.dmi'
-
+		// var/icon_file = 'icons/mob/clothing/id.dmi' SKYRATEDIT - PDA-B-GONE
+/* SKYRAT REMOVAL BEGIN - PDA-B-GONE
 		id_overlay = wear_id.build_worn_icon(default_layer = ID_LAYER, default_icon_file = icon_file)
```

Comment:
This should be all you need.
```suggestion
		/* SKYRAT EDIT REMOVAL - PDA-B-GONE
		id_overlay = wear_id.build_worn_icon(default_layer = ID_LAYER, default_icon_file = icon_file)
		*/ // SKYRAT EDIT END
```

Reply:
that did work, I probably just did it wrong the last time i tested